test/governance/Governor.test.js

```diff
@@ -18,24 +16,47 @@ const Governor = artifacts.require('GovernorMock');
 const CallReceiver = artifacts.require('CallReceiverMock');
 
 contract('Governor', function (accounts) {
+  const helper = new GovernorHelper();
+
   const [ owner, proposer, voter1, voter2, voter3, voter4 ] = accounts;
+  const empty = web3.utils.toChecksumAddress(web3.utils.randomHex(20));
 
   const name = 'OZ-Governor';
   const version = '1';
   const tokenName = 'MockToken';
   const tokenSymbol = 'MTKN';
   const tokenSupply = web3.utils.toWei('100');
+  const votingDelay = new BN(4);
+  const votingPeriod = new BN(16);
+  const value = web3.utils.toWei('1');
 
   beforeEach(async function () {
-    this.owner = owner;
+    this.chainId = await web3.eth.getChainId();
     this.token = await Token.new(tokenName, tokenSymbol);
-    this.mock = await Governor.new(name, this.token.address, 4, 16, 10);
+    this.mock = await Governor.new(name, this.token.address, votingDelay, votingPeriod, 10);
     this.receiver = await CallReceiver.new();
+
+    await web3.eth.sendTransaction({ from: owner, to: this.mock.address, value });
+
     await this.token.mint(owner, tokenSupply);
     await this.token.delegate(voter1, { from: voter1 });
     await this.token.delegate(voter2, { from: voter2 });
     await this.token.delegate(voter3, { from: voter3 });
     await this.token.delegate(voter4, { from: voter4 });
+    await this.token.transfer(voter1, web3.utils.toWei('10'), { from: owner });
+    await this.token.transfer(voter2, web3.utils.toWei('7'), { from: owner });
+    await this.token.transfer(voter3, web3.utils.toWei('5'), { from: owner });
+    await this.token.transfer(voter4, web3.utils.toWei('2'), { from: owner });
```

Comment:
Did you consider putting this in the helper?

Reply:
the helper is not token aware ... 

I'm not sure it would make things clearer, because even if the helper knew about the token, you'd still need one line per allocation, right ?
Also, not sure how NFT would be handeled